DOC-2578: Scroll to show action buttons when replying/editing a comment. (#3523)
tinymce/tinymce-docs: modules/ROOT/pages/7.6.0-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== Comments

The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.

**Comments** Premium plugin includes the following fixes and improvements.

> ==== Always show the comment input field in the viewport when pressing the reply or add comment button

Reviewer: ```suggestion
==== Scroll to show action buttons when replying/editing a comment.
```

> Previously, if a selected conversation card was positioned near the bottom of the sidebar, the reply input field or edit input field would be below the bottom of the sidebar resulting in the comment input field not being visible to the user.

Reviewer: ```suggestion
Previously, if a selected conversation card was positioned near the bottom of the sidebar, the reply/edit input field would be below the bottom of the sidebar resulting in the comment input field not being visible to the user.
```

> With the release of {productname} {release-version}, the sidebar now scrolls to display the comment input field if the selected conversation card is near the bottom of the sidebar. This ensures that the comment input field is always visible to the user.

Reviewer: ```suggestion
{productname} {release-version} addresses this issue. Now, the sidebar scrolls to display the comment input field if the selected conversation card is positioned near the bottom of the sidebar. This ensures that the comment input field is always visible to the user.
```

[[improvements]]
== Improvements

{productname} {release-version} also includes the following improvements:

=== Alway show the comment input field in the viewport when pressing the reply or add comment button
// #TINY-11402

Previously, if a selected conversation card was positioned near the bottom of the sidebar, the reply input field or edit input field would be below the bottom of the sidebar resulting in the comment input field not being visible to the user.

> With the release of {productname} {release-version}, the sidebar now scrolls to display the comment input field. This ensures that the comment input field is always visible to the user.

Reviewer 2: I think we should mention that we only scroll in the above mentioned situation. Because reading this sentence as it is, I understand that now we always scroll.

Author: @hamza0867 I've pushed another commit with your suggestion, let me know if we should add anything else.
